[BUG] Popups throw exception in MAUI 9.0.50

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Did you read the "Reporting a bug" section on Contributing file?

- [x] I have read the "Reporting a bug" section on Contributing file: https://github.com/CommunityToolkit/Maui/blob/main/CONTRIBUTING.md#reporting-a-bug

### Current Behavior

When trying to open a CommunityToolkit popup the following exception is thrown.

System.MissingMethodException: 'Method not found: void Microsoft.Maui.Controls.Internals.PropertyPropagationExtensions.PropagatePropertyChanged(string,Microsoft.Maui.Controls.Element,System.Collections.IEnumerable)'

From WinUI:

![Image](https://github.com/user-attachments/assets/4b268a80-64e2-41a4-89fe-72a83a9d9c2e)

### Expected Behavior

The CommunityToolkit popup should open with no exception thrown.

### Steps To Reproduce

1. Update MAUI packages to 9.0.50
2. Open a CommunityToolkit popup

### Link to public reproduction project repository

https://github.com/unrateddi/repro-community-toolkit-popup-maui-9.0.50-crash

### Environment

```markdown
- .NET MAUI CommunityToolkit: 11.1.0
- OS:
- .NET MAUI: 9.0.50
```

### Anything else?

_No response_

getting the same thing now

[mono-rt] [ERROR] FATAL UNHANDLED EXCEPTION: System.MissingMethodException: Method not found: void Microsoft.Maui.Controls.Internals.PropertyPropagationExtensions.PropagatePropertyChanged(string,Microsoft.Maui.Controls.Element,System.Collections.IEnumerable)
[mono-rt]    at Microsoft.Maui.Controls.Element.OnParentSet() in /_/src/Controls/src/Core/Element/Element.cs:line 631
[mono-rt]    at Microsoft.Maui.Controls.Element.SetParent(Element value) in /_/src/Controls/src/Core/Element/Element.cs:line 430
[mono-rt]    at Microsoft.Maui.Controls.Element.OnChildAdded(Element child) in /_/src/Controls/src/Core/Element/Element.cs:line 594
[mono-rt]    at Microsoft.Maui.Controls.VisualElement.OnChildAdded(Element child) in /_/src/Controls/src/Core/VisualElement/VisualElement.cs:line 1310
[mono-rt]    at Microsoft.Maui.Controls.Element.AddLogicalChild(Element element) in /_/src/Controls/src/Core/Element/Element.cs:line 225
[mono-rt]    at CommunityToolkit.Maui.Views.PopupExtensions.CreatePopup(Page page, Popup popup) in /_/src/CommunityToolkit.Maui/Views/Popup/PopupExtensions.shared.cs:line 132
[mono-rt]    at CommunityToolkit.Maui.Views.PopupExtensions.CreateAndShowPopup[BarcodeScanner](Page page, BarcodeScanner popup) in /_/src/CommunityToolkit.Maui/Views/Popup/PopupExtensions.shared.cs:line 149
[mono-rt]    at CommunityToolkit.Maui.Views.PopupExtensions.ShowPopup[BarcodeScanner](Page page, BarcodeScanner popup) in /_/src/CommunityToolkit.Maui/Views/Popup/PopupExtensions.shared.cs:line 37
[mono-rt]    at DSDRouteManager.Controls.DSDBarcodeScanner.<>c__DisplayClass4_0.<ScanBarcodeAsync>b__0() in C:\Users\james\source\repos\Maui\DSDRouteManager\DSDRouteManager\Controls\DSDBarcodeScanner.cs:line 32

--- a/Directory.Build.props
+++ b/Directory.Build.props
@@ -16,7 +16,7 @@
     <NuGetAuditMode>all</NuGetAuditMode>
 
     <!-- MAUI Specific -->
-    <MauiPackageVersion>9.0.40</MauiPackageVersion>
+    <MauiPackageVersion>9.0.50</MauiPackageVersion>
     <NextMauiPackageVersion>10.0.0</NextMauiPackageVersion>
     <MauiStrictXamlCompilation>true</MauiStrictXamlCompilation>
     <SkipValidateMauiImplicitPackageReferences>true</SkipValidateMauiImplicitPackageReferences>

